server/src/main/java/org/opensearch/cluster/routing/remote/InternalRemoteRoutingTableService.java

```diff
@@ -152,42 +113,34 @@ public DiffableUtils.MapDiff<String, IndexRoutingTable, Map<String, IndexRouting
     /**
      * Create async action for writing one {@code IndexRoutingTable} to remote store
      * @param clusterState current cluster state
+     * @param clusterUUID current cluster UUID
```

Comment:
Do we plan on adding clusterUUID to this method param?

Reply:
Fixed the java docs as per the params